### PR TITLE
Add credit card options

### PR DESCRIPTION
**Pull request by adamnoto**
Allow users to set credit card options for `VtWebParam`

### CODE REVIEW

**Comment by adamnoto** [comment]
@robeth or @drabiter kindly review :)

**Comment by drabiter** [comment]
LGTM :ok_hand:

**Comment by robeth** [comment]
Nice mas adam :+1: 

The channel options implementation for vtweb is :ok_hand:

File: src/main/java/id/co/veritrans/mdk/v1/gateway/model/vtweb/CreditCardOptions.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package id.co.veritrans.mdk.v1.gateway.model.vtweb;
+
+import id.co.veritrans.mdk.v1.gateway.model.vtdirect.paymentmethod.CreditCard;
+
+/**
+ * Set the credit card option such as the channel (eg. MIGS).
+ * Created by adampahlevi on 12/29/16.
```
Comment (robeth) [inline src/main/java/id/co/veritrans/mdk/v1/gateway/model/vtweb/CreditCardOptions.java]: Not sure about convention in this project. But its recommended to turn author signature off in Intellij/Eclipse

Your contribution is already carved in history via git :)

Comment (adamnoto) [reply src/main/java/id/co/veritrans/mdk/v1/gateway/model/vtweb/CreditCardOptions.java]: sorry I didn't know we don't need to submit that. certainly.

no worry.

**Pull request merged**